test/jdk/java/lang/invoke/WrongMethodTypeTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
```

Comment:
From what I understand, the copyright notice on test files don't use the "Classpath" exception and instead use a different copyright notice.

Reply:
That's correct.   Thanks for catching it.   This reveals that the copyright header of several `test/jdk/java/lang/invoke` tests will need clean up (as this new test was copied from an existing test).